packages/react-native/src/index.ts

```diff
@@ -19,29 +19,29 @@ const BreezSDKLiquid = NativeModules.RNBreezSDKLiquid
 
 const BreezSDKLiquidEmitter = new NativeEventEmitter(BreezSDKLiquid)
 
-export interface AesSuccessActionDataDecrypted {
+export type AesSuccessActionDataDecrypted = {
```

Comment:
Seems to be producing old syntax. What's your `tslint` and `typescript` versions?
```
tslint --version
tsc --version
```
I have:
```
tslint 6.1.3
tsc 5.4.5
```

Reply:
```
tslint: 6.1.3
tsc: 5.5.4
```

Reply:
Unless I'm missing something CI used the exact same versions on my env, but running the codegen does not introduce changes for me.

```
tslint: 6.1.3
typescript: 5.5.4
kotlin: 2.0.10 
swiftformat: 0.54.3
ktlint: 1.3.1
```

Reply:
The difference was on `packages/react-native`. Running `make all` on `packages/react-native/` after fixing Payment on UDL file has addressed this & `cargo clippy` issues.

- [Update Payment on udl file](https://github.com/breez/breez-sdk-liquid/pull/414/commits/a7d43d8a77e7a2510981d5bdbd98b4d7d1fce14e)
- [chore: run 'make all' on 'packages/react-native/'](https://github.com/breez/breez-sdk-liquid/pull/414/commits/fe981e6cf435a9fa8d2daf1743a025b04673913c)